shard.yml

```diff
@@ -1,8 +1,8 @@
 name: granite
 
-version: 0.22.0
+version: 1.0.0
 
-crystal: 0.35.0
+crystal: 1.0.0
```

Comment:
Does the version specifier need to be more specific here? The semantics of the crystal version comparison [aren't quite what most people expect](https://github.com/crystal-lang/shards/blob/master/docs/shard.yml.adoc)

Reply:
updated, thanks